.github/workflows/e2e-airgap.yml

```
@@ -85,6 +85,22 @@ jobs:
           cd ${GITHUB_WORKSPACE}/e2e-tests/tests/airgap
           make e2e-prepare-archive
 
+      - name: Extract component versions/informations
+        id: component
+        run: |
+          HELM_FOLDER=/home/gh-runner/airgap_rancher/helm
+          KUBEWARDEN_CONTROLLER_IMG_FILE=${HELM_FOLDER}/kubewarden-controller/imagelist.txt
+          KUBEWARDEN_DEFAULTS_IMG_FILE=${HELM_FOLDER}/kubewarden-defaults/imagelist.txt
+
+          AUDIT_SCANNER_VERSION=$(grep 'audit-scanner' ${KUBEWARDEN_CONTROLLER_IMG_FILE} | cut -d':' -f2)
+          KUBEWARDEN_CONTROLLER_VERSION=$(grep 'kubewarden-controller' ${KUBEWARDEN_CONTROLLER_IMG_FILE} | cut -d':' -f2)
+          POLICY_SERVER_VERSION=$(grep 'policy-server' ${KUBEWARDEN_DEFAULTS_IMG_FILE} | cut -d':' -f2)
+
+          # Export values
+          echo "audit_scanner_version=${AUDIT_SCANNER_VERSION}" >> ${GITHUB_OUTPUT}
```

I usually use this to see values github log:
```suggestion
          echo "audit_scanner_version=${AUDIT_SCANNER_VERSION}" | tee -a ${GITHUB_OUTPUT}
```

---

For Elemental CI, we use `>>`everywhere but it you use `tee` in Kubewarden CI, let's switch to it then.